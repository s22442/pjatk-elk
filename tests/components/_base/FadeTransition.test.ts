import FadeTransition from '~/components/_base/FadeTransition.vue';

import { mount } from '@vue/test-utils';

describe('FadeTransition', () => {
  it('should render', () => {
    const wrapper = mount(FadeTransition, {
      slots: { default: '<div> abc </div>' },
    });

    expect(wrapper.html()).toContain('<div> abc </div>');
    expect(wrapper.html()).toMatchSnapshot();
  });
});
