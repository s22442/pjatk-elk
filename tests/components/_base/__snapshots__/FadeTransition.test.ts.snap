// Vitest Snapshot v1

exports[`FadeTransition > should render 1`] = `
"<transition-stub enterfromclass=\\"op0\\" leavetoclass=\\"op0\\" enteractiveclass=\\"transition-opacity pointer-events-none\\" leaveactiveclass=\\"transition-opacity pointer-events-none\\" mode=\\"out-in\\" appear=\\"false\\" persisted=\\"false\\" css=\\"true\\">
  <div> abc </div>
</transition-stub>"
`;
